pkg/controller/service/reconciler.go

```diff
@@ -354,7 +354,7 @@ func (s *ServiceReconciler) defaultInventoryObjTemplate(id string) (*unstructure
 	}, nil
 }
 
-func disableClientLimits(config *rest.Config) {
+func DisableClientLimits(config *rest.Config) {
```

Comment:
Can you export it to some common/utils package since it seems to be used in multiple packages now?